test_package/pylake/test.py

```diff
@@ -0,0 +1,5 @@
+import sys
+import lumicks.pylake as lk
+
+lk.benchmark()
+sys.exit()
```

Comment:
No need for `sys.exit()` here. It's the end of the script anyway.
```suggestion
import lumicks.pylake as lk

lk.benchmark()
```